### ISSUE TITLE
Display hyperlink URLs in the Status Bar

### ISSUE THREAD

**Issue by eboracus** [atom/status-bar#89]
It would be nice if there was an option for the Status Bar to display the URL of a link when hovering over it with the mouse. Mimicking standard Browser behaviour. I'm mainly referring to links displayed in the Settings/Packages etc. views as opposed to any full URLs already displayed in the text editor view itself.

![hyperlink_statusbar](https://cloud.githubusercontent.com/assets/11758299/8464236/ab21454a-2038-11e5-9beb-293a2f9a7cbe.png)

Links on the titles and author's usernames of packages (for ex.) are, of course, obvious as to where they point because there is a set convention for what they lead to. But the primary usefulness of this feature I think would be for the more general 'in page' links like those shown above. Plus, in the Settings view at least, there should be plenty of available free space for display.

**Comment by mnquintana** [comment]
This seems like a great idea for a package.

**Comment by benogle** [comment]
Agreed. I would love to see this. I think a package would be awesome. I could see us bundling something like this as well.

**Comment by mnquintana** [comment]
This would also essentially resolve https://github.com/atom/settings-view/issues/330

**Comment by eboracus** [comment]
I just came across #330 and was going to link here... until I reached the bottom. :) I think a URL down at the bottom like this would certainly be better than the one appearing by the cursor.

I'd give this a go myself, but I use other languages and only relatively new into Less/CoffeeScript, so something of this kind is currently out of my scope. If this is still open further down the line though, I can certainly try it.
